### PR TITLE
Added missing imports

### PR DESCRIPTION
**Pull request by Noooil**
[//]: # (**********************************)
[//]: # (** Fill in the following fields **)
[//]: # (**********************************)

[//]: # (Note: Lines beginning with syntax such as this one, are comments and will not be visible in your report!)

### Pull Request Prelude

[//]: # (Thank you for working on improving Hercules!)

[//]: # (Please complete these steps and check the following boxes by putting an `x` inside the brackets _before_ filing your Pull Request.)

- [X] I have followed [proper Hercules code styling][code].
- [X] I have read and understood the [contribution guidelines][cont] before making this PR.
- [X] I am aware that this PR will be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

[//]: # (Describe at length, the changes that this pull request makes.)
It adds some missing imports so you can configure your hercules 100% just by using the import folder.

**Affected Branches:** 

[//]: # (Master? Slave?)
master

**Issues addressed:**

[//]: # (Issue Tracker Number if any.)

### Known Issues and TODO List

[//]: # (Insert checklist here)
[//]: # (Syntax: - [ ] Checkbox)

[//]: # (**NOTE** Enable the setting "[√] Allow edits from maintainers." when creating your pull request if you have not already enabled it.)

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

File: conf/global/sql_connection.conf [hunk 1]
```diff
@@ -50,3 +50,6 @@ sql_connection: {
 	db_database: "ragnarok"
 	//codepage:""
 }
+
+// Your custom config goes here.
+import: "conf/import/sql_connection.conf"
```
Comment (dastgir) [inline conf/global/sql_connection.conf]: I won't recommend this,
Just see how map/char/login server include sql_connection, and follow same structure in their imports.

Comment (Noooil) [reply conf/global/sql_connection.conf]: Should I update the comment to make it more clear? In my case I copy the same settings to 3 files that doesnt feel right

Comment (MishimaHaruna) [reply conf/global/sql_connection.conf]: I don't believe we look for an `import` key in this context - if we were do to this, the source would need to be modified accordingly, or am I missing something?

**Comment by Noooil** [comment]
Need to edit sources first

**Pull request closed without merging**